src/components/Refero/ReferoPreview.tsx

```diff
@@ -25,7 +25,7 @@ type Props = {
 // eslint-disable-next-line @typescript-eslint/ban-types
 const store: Store<{}> = createStore(rootReducer, applyMiddleware(thunk));
 
-const ReferoFiller = ({ showFormFiller, language }: Props): JSX.Element => {
+const ReferoPreview = ({ showFormFiller, language }: Props): JSX.Element => {
```

Comment:
Or just use FormFillerPreview?

Reply:
Ok, I can rename the component to FormFillerPreview